tests/plugins/optimization-detective/optimization-tests.php

```diff
@@ -178,158 +178,6 @@ public function test_od_can_optimize_response( Closure $set_up, bool $expected )
-							'srcset'      => 'https://example.com/elva-fairy-480w.jpg 480w, https://example.com/elva-fairy-800w.jpg 800w',
-							'sizes'       => '(max-width: 600px) 480px, 800px',
-							'crossorigin' => 'anonymous',
-						),
-					),
-					601 => array(
-						'img_attributes' => array(
-							'src'         => 'https://example.com/alt-elva-fairy-800w.jpg',
-							'srcset'      => 'https://example.com/alt-elva-fairy-480w.jpg 480w, https://example.com/alt-elva-fairy-800w.jpg 800w',
-							'sizes'       => '(max-width: 600px) 480px, 800px',
-							'crossorigin' => 'anonymous',
-						),
-					),
-				),
-				'expected'                                => '
-					<link data-od-added-tag rel="preload" fetchpriority="high" as="image" href="https://example.com/elva-fairy-800w.jpg" imagesrcset="https://example.com/elva-fairy-480w.jpg 480w, https://example.com/elva-fairy-800w.jpg 800w" imagesizes="(max-width: 600px) 480px, 800px" crossorigin="anonymous" media="screen and (max-width: 600px)">
-					<link data-od-added-tag rel="preload" fetchpriority="high" as="image" href="https://example.com/alt-elva-fairy-800w.jpg" imagesrcset="https://example.com/alt-elva-fairy-480w.jpg 480w, https://example.com/alt-elva-fairy-800w.jpg 800w" imagesizes="(max-width: 600px) 480px, 800px" crossorigin="anonymous" media="screen and (min-width: 601px)">
-				',
-			),
-			'two-non-consecutive-responsive-lcp-images' => array(
-				'lcp_elements_by_minimum_viewport_widths' => array(
-					0   => array(
-						'img_attributes' => array(
-							'src'         => 'https://example.com/elva-fairy-800w.jpg',
-							'srcset'      => 'https://example.com/elva-fairy-480w.jpg 480w, https://example.com/elva-fairy-800w.jpg 800w',
-							'sizes'       => '(max-width: 600px) 480px, 800px',
-							'crossorigin' => 'anonymous',
-						),
-					),
-					481 => false,
-					601 => array(
-						'img_attributes' => array(
-							'src'         => 'https://example.com/alt-elva-fairy-800w.jpg',
-							'srcset'      => 'https://example.com/alt-elva-fairy-480w.jpg 480w, https://example.com/alt-elva-fairy-800w.jpg 800w',
-							'sizes'       => '(max-width: 600px) 480px, 800px',
-							'crossorigin' => 'anonymous',
-						),
-					),
-				),
-				'expected'                                => '
-					<link data-od-added-tag rel="preload" fetchpriority="high" as="image" href="https://example.com/elva-fairy-800w.jpg" imagesrcset="https://example.com/elva-fairy-480w.jpg 480w, https://example.com/elva-fairy-800w.jpg 800w" imagesizes="(max-width: 600px) 480px, 800px" crossorigin="anonymous" media="screen and (max-width: 480px)">
-					<link data-od-added-tag rel="preload" fetchpriority="high" as="image" href="https://example.com/alt-elva-fairy-800w.jpg" imagesrcset="https://example.com/alt-elva-fairy-480w.jpg 480w, https://example.com/alt-elva-fairy-800w.jpg 800w" imagesizes="(max-width: 600px) 480px, 800px" crossorigin="anonymous" media="screen and (min-width: 601px)">
-				',
-			),
```

Comment:
Test case already handled by `test_od_optimize_template_output_buffer` via `different-lcp-elements-for-non-consecutive-viewport-groups-with-missing-data-for-middle-group`.